src/plugins/blocksmith/plugins/focus/index.js

```diff
@@ -196,14 +151,11 @@ export default (config = {}) => {
     // Handle down/up arrow events and set activeBlock/selection if necessary
     onDownArrow: (event, { getEditorState, setEditorState }) => {
       // TODO edgecase: if one block is selected and the user wants to expand the selection using the shift key
-
+      
       const editorState = getEditorState();
-      const lines = getLines(editorState, testClass);
-      let offsetAtStartOfLastLine = 0;
-      for (let i = 0; i < lines.length - 1; i += 1) {
-        offsetAtStartOfLastLine += lines[i].length;
-      }
-      if (lines.length <= 1 || editorState.getSelection().getAnchorOffset() + 1 > offsetAtStartOfLastLine) {
+      const lastLine = isOnLastLineOfBlock(editorState, testClass);
+      
+      if (lastLine) {      
```

Comment:
@nanetteCodes just put the function call inside the if. then it reads almost like a normal human sentence "if is on last line of block"